class Queueable:
    run_again_on_fail: bool
    run_times: int
    def handle(self) -> None: ...
    def failed(self, obj: object, e: Exception) -> None: ...
